projects/configs/tag-operations/mri.yaml

```diff
@@ -166,22 +162,42 @@
   group: 0x0018
   element: 0x0083
   op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180083
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189119/00180083
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189119/00180083
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189119/00180083
 - name: "Magnetic Field Strength"
   group: 0x0018
   element: 0x0087
   op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180087
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image/00180087
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-image/00180087
+#https://dicom.innolitics.com/ciods/legacy-converted-enhanced-mr-image/enhanced-mr-image/00180087
 - name: "Spacing Between Slices"
   group: 0x0018
   element: 0x0088
   op: "keep"
+#https://dicom.innolitics.com/ciods/ct-image/image-plane/00180088
+#https://dicom.innolitics.com/ciods/mr-image/image-plane/00180088
+#https://dicom.innolitics.com/ciods/x-ray-3d-angiographic-image/x-ray-3d-angiographic-image-multi-frame-functional-groups/52009229/00289110/00180088
+#https://dicom.innolitics.com/ciods/x-ray-3d-craniofacial-image/x-ray-3d-craniofacial-image-multi-frame-functional-groups/52009229/00289110/00180088
 - name: "Echo train length"
   group: 0x0018
   element: 0x0091
   op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180091
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189112/00180091
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009229/00189112/00180091
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189112/00180091
 - name: "Percent Phase Field of View"
   group: 0x0018
   element: 0x0094
   op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180094
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189103/00180094
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009229/00189125/00180094
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189125/00180094
 - name: "Software Version"
   group: 0x0018
   element: 0x1020
```

Comment:
[50cd1e3](https://github.com/SAFEHR-data/PIXL/pull/498/commits/50cd1e3e5b7a467dc062ffc87fb1848983dfa48e)

Reply:
[removing tags from mri that are already part of base.yml would break …get_private_tag; (see CI when removing Private Creator Data Elemen); do we want to do that? ](https://github.com/SAFEHR-data/PIXL/pull/498/commits/f34efe365253efcaad1f38c1eef32c12780c3fdb)  [f34efe3](https://github.com/SAFEHR-data/PIXL/pull/498/commits/f34efe365253efcaad1f38c1eef32c12780c3fdb)